controllers/operands/operandHandler.go

```diff
@@ -67,6 +66,12 @@ func NewOperandHandler(client client.Client, scheme *runtime.Scheme, isOpenshift
 		}...)
 	}
 
+	if ci.IsOpenshift() && ci.IsConsolePluginImageProvided() {
+		operands = append(operands, []Operand{
+			newConsoleHandler(client),
+		}...)
+	}
+
```

Comment:
Not important: can be embedded within the previous condition. e.g. 
```suggestion
	if ci.IsOpenshift() {
		operands = append(operands, []Operand{
			newSspHandler(client, scheme),
			(*genericOperand)(newMetricsServiceHandler(client, scheme)),
			(*genericOperand)(newMetricsServiceMonitorHandler(client, scheme)),
			(*genericOperand)(newMonitoringPrometheusRuleHandler(client, scheme)),
			(*genericOperand)(newCliDownloadHandler(client, scheme)),
			(*genericOperand)(newCliDownloadsRouteHandler(client, scheme)),
			(*genericOperand)(newCliDownloadsServiceHandler(client, scheme)),
			newNamespaceHandler(client, scheme),
		}...)

	        if ci.IsConsolePluginImageProvided() {
		    operands = append(operands, []Operand{
			    newConsoleHandler(client),
		    }...)
		}
	}

```

But we can do it in a separate PR.